app/models/user.rb

```diff
@@ -0,0 +1,26 @@
+require 'data_mapper'
+require 'bcrypt'
+require 'dm-validations'
+require 'dm-postgres-adapter'
+require 'dm-migrations'
+
+class User
+  include DataMapper::Resource
+
+  property :id, Serial
+  property :name, String
+  property :username, String, :unique => true
```

Comment:
Style/HashSyntax: Use the new Ruby 1.9 hash syntax. (https://github.com/bbatsov/ruby-style-guide#hash-literals)